public/app/features/datasources/components/DataSourceTypeCard.tsx

```diff
@@ -50,7 +51,7 @@ export function DataSourceTypeCard({ onClick, dataSourcePlugin }: Props) {
             icon="external-link-alt"
```

Comment:
I think I could change the icon here, but don't know if we have any convention

Reply:
Yeah, I am not sure either which icon we would / should use instead. Would it make sense to just not use an icon or to use something like `plus`?

Reply:
Removed the icon completely https://github.com/grafana/grafana/pull/63499/commits/354a8c6f238f91977dc69c8e0de46fa202c19ca1